### ISSUE TITLE
after upgrading Flutter to 1.3.8, connectivity plugin has runtimeExceptions

### ISSUE THREAD

**Issue by tb2761** [flutter/flutter#29877]
## Steps to Reproduce

1. create new flutter app
2. add connectivity package
3. get or listen on conectivity

## Logs

```
java.lang.RuntimeException: 
  at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0 (LoadedApk.java:1493)
  at android.app.-$$Lambda$LoadedApk$ReceiverDispatcher$Args$_BumDX2UKsnxLVrE6UJsJZkotuA.run (Unknown Source:2)
  at android.os.Handler.handleCallback (Handler.java:873)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:214)
  at android.app.ActivityThread.main (ActivityThread.java:7032)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:493)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:964)
Caused by: java.lang.RuntimeException: 
  at io.flutter.embedding.engine.FlutterJNI.ensureAttachedToNative (FlutterJNI.java:514)
  at io.flutter.embedding.engine.FlutterJNI.dispatchPlatformMessage (FlutterJNI.java:444)
  at io.flutter.embedding.engine.dart.DartMessenger.send (DartMessenger.java:74)
  at io.flutter.embedding.engine.dart.DartExecutor.send (DartExecutor.java:149)
  at io.flutter.view.FlutterNativeView.send (FlutterNativeView.java:142)
  at io.flutter.plugin.common.EventChannel$IncomingStreamRequestHandler$EventSinkImplementation.success (EventChannel.java:213)
  at io.flutter.plugins.connectivity.ConnectivityPlugin$1.onReceive (ConnectivityPlugin.java:143)
  at android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0 (LoadedApk.java:1483)
```

```
[v] Flutter (Channel beta, v1.3.8, on Microsoft Windows [Version 10.0.17763.379], locale de-DE)
    • Flutter version 1.3.8 at C:\Users\Tim\Flutter\src\flutter
    • Framework revision e5b1ed7a7f (3 weeks ago), 2019-03-06 14:23:37 -0800
    • Engine revision f4951df193
    • Dart version 2.2.1 (build 2.2.1-dev.0.0 571ea80e11)

[v] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at C:\Users\Tim\AppData\Local\Android\Sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = C:\Users\Tim\AppData\Local\Android\Sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[v] Android Studio (version 3.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin version 33.4.1
    • Dart plugin version 182.5215
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[v] VS Code (version 1.32.3)
    • VS Code at C:\Users\Tim\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 2.24.0

[!] Connected device
    ! No devices available

! Doctor found issues in 1 category.

```

**Comment by fazil-augr** [comment]
I am using flutter 'flutter_offline' same error is happening to me. In my play store console the same error is shown in crash report. It is effected in android version 9.

**Comment by tb2761** [comment]
fixed on Stable Version 1.5.4-hotfix.2.
